src/platform/packages/shared/kbn-server-route-repository/src/register_routes.ts

```diff
@@ -43,11 +43,13 @@ export function registerRoutes<TDependencies extends Record<string, any>>({
   repository,
   logger,
   dependencies,
+  validateSchemas,
 }: {
   core: CoreSetup;
   repository: Record<string, ServerRoute<string, RouteParamsRT | undefined, any, any, any>>;
   logger: Logger;
   dependencies: TDependencies;
+  validateSchemas: boolean;
```

Comment:
I think it would be better to use `isDev` instead of `validateSchemas.` The current naming suggests that the whole validation is skipped, but it is just extra dev validation that is ignored.
Also, for a new developer, it would not be clear when to set this field.

Reply:
Fair point, I've renamed it to `runDevModeChecks`